data/zilliqa/assets/zil1fam2m6a0f2mv2hjys0rdzr4sm5drryt99w9c4a/meta.json

```diff
@@ -1,21 +1,20 @@
 {
-  "name": "TYRON Token",
+  "name": "Tyron Self-Sovereign Identity (SSI) Token",
```

Comment:
That name is too big, change to "Tyron SSI Token"

Reply:
hi @balthazar, thanks for the feedback. I've sent the commit with the shorter name

Reply:
hello team, gm @balthazar, thanks for merging my PR.

https://viewblock.io/zilliqa/address/0x4f76adebaf4ab6c55e4483c6d10eb0dd1a319165 is not showing updated data yet - tho our smart contract is fully compatible with the ZRC2 interface. Can you help me double-check? Happy to chat with any team member to share more info if that would help.

kind regards,
Xalkan

Reply:
deployed something else, ping me again if it's not reflected in the next 1-2h